js/constants/constants.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ERROR_SCREEN: "errorScreen"
   },
 
+  CUSTOM_EVENTS: {
```

Comment:
I used custom events in order to avoid adding a dependent of `INITIAL_PLAY`, in case this caused any issues. The regular event's name is `INITIAL_PLAY` though, do you want to change it to `DEFERRED_INITIAL_PLAY`?